x-pack/plugins/dataset_quality/public/components/dataset_quality/columns.tsx

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiBasicTableColumn, EuiFlexGroup, EuiFlexItem, EuiText } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { PackageIcon } from '@kbn/fleet-plugin/public';
+import { Integration } from '../../../common/data_streams_stats/integration';
+import { DataStreamStat } from '../../../common/data_streams_stats/data_stream_stat';
+import { useFormattedTime } from '../../hooks';
+
+const nameColumnName = i18n.translate('xpack.datasetQuality.nameColumnName', {
+  defaultMessage: 'Dataset Name',
+});
+
+const integrationColumnName = i18n.translate('xpack.datasetQuality.integrationColumnName', {
+  defaultMessage: 'Integration',
+});
+
+const sizeColumnName = i18n.translate('xpack.datasetQuality.sizeColumnName', {
+  defaultMessage: 'Size',
+});
+
+const lastActivityColumnName = i18n.translate('xpack.datasetQuality.lastActivityColumnName', {
+  defaultMessage: 'Last Activity',
+});
+
+export const getDatasetQualitTableColumns = (): Array<EuiBasicTableColumn<DataStreamStat>> => {
+  return [
+    {
+      name: nameColumnName,
+      field: 'title',
+      sortable: true,
+      width: '400px',
```

Comment:
should this be a custom renderer as well?
at least I would use `name` that is the raw dataset name. Instead I'm a seeing
<img width="355" alt="image" src="https://github.com/elastic/kibana/assets/1313018/345f6018-763c-44ab-81d4-23cdfee8c27c">